src/__experimental__/components/grouped-character/index.js

```diff
@@ -0,0 +1,3 @@
+import GroupedCharacter from './grouped-character.component';
```

Comment:
Would you like to export component without import syntax?

Reply:
changed